### PR TITLE
feat: dynamically render application custom questions

### LINKED ISSUE #535: Update application details page for custom questions
**User Story:**
As a round operator 
I want to see the questions I asked my applicants on the application detail page
So that I can properly assess my grantees

**Acceptance Criteria**
- application details page is no longer hard-coded
- application details page renders using dynamic application questions
- move email field into dynamic application questions (instead of icon in subheader)

**Product & Design Links:**

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

### PR DESCRIPTION
**Pull request by chibie**
##### Description

This pr dynamically renders the custom questions in the application detail page

##### Refers/Fixes

closes #535

### CODE REVIEW

File: packages/contracts/package.json
Comment (thelostone-mc) [inline packages/contracts/package.json:15]: Any reason for this change ?

**Review by thelostone-mc — COMMENTED** [review]
left a question but otherwise LGTM

**Comment by melissa-neira** [comment]
Is anyone else getting this pale purple color on the project description page in GE?
screencapture-gegitcoin-on-fleek-co-2022-11-01-10_16_22

**Review by nfrgosselin — APPROVED** [review]

File: packages/contracts/package.json
Comment (chibie) [reply packages/contracts/package.json:15]: the docs specify `yarn run lint:sol` for solidity and `yarn run lint:ts` for typescript

**Comment by thelostone-mc** [comment]
@melissa-neira Just checked ! It looks alright to me

**Comment by nfrgosselin** [comment]
@melissa-neira this one is good to merge once you approve :)

**Review by melissa-neira — APPROVED** [review]

**Pull request merged**
